src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'underscore';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Pressable} from 'react-native';
+
+const propTypes = {
+    // The function that should be called when this pressable is LongPressed or right-clicked.
+    onSecondaryInteraction: PropTypes.func.isRequired,
+
+    // The children which should be contained in this wrapper component.
+    children: PropTypes.node.isRequired,
+};
+
+/**
+ * This is a special Pressable that calls onSecondaryInteraction when LongPressed, or right-clicked.
+ */
+class PressableWithSecondaryInteraction extends Component {
+    constructor(props) {
+        super(props);
+
+        this.executeSecondaryInteractionOnContextMenu = this.executeSecondaryInteractionOnContextMenu.bind(this);
+    }
+
+    componentDidMount() {
+        this.pressableRef.addEventListener('contextmenu', this.executeSecondaryInteractionOnContextMenu);
+    }
+
+    componentWillUnmount() {
+        this.pressableRef.removeEventListener('contextmenu', this.executeSecondaryInteractionOnContextMenu);
+    }
+
+    /**
+     * @param {contextmenu} e - A contextmenu event.
```

Comment:
NAB, should this be `ContextMenuEvent`? Not sure if it's the case...

Reply:
oh actually I think contextmenu is a kind of `MouseEvent`

Reply:
![Screen Shot 2021-01-25 at 12 14 52 PM](https://user-images.githubusercontent.com/32969087/105772882-f59bc100-5f06-11eb-8e59-11a379ff5024.png)